assets/js/custom/Follower.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-env jquery */
+/* global Swal */
+
+// eslint-disable-next-line no-unused-vars
+function Follower (unfollowUrl, followUrl, somethingWentWrongError, followError, unfollowError, unfollowButton, unfollowQuestion, cancelButton,
+  visibleFollowing = 5, visibleFollowers = 5, showSte = 5,
+  minAmountOfVisibleFollowers = 5, totalFollowing = 5, totalFollowers = 5) {
+  const self = this
+  self.unfollowUrl = unfollowUrl
+  self.followUrl = followUrl
+  self.somethingWentWrongError = somethingWentWrongError
+  self.followError = followError
+  self.unfollowError = unfollowError
+  self.unfollowButton = unfollowButton
+  self.unfollowQuestion = unfollowQuestion
+  self.cancelButton = cancelButton
+
+  self.unfollow = function (id, username) {
+    const $followerItems = $('.follower-item-' + id)
+    const $buttons = $followerItems.find('.btn-follow button').attr('disabled', true)
+
+    Swal.fire({
+      title: self.unfollowQuestion,
+      icon: 'question',
+      showCancelButton: true,
+      confirmButtonText: self.unfollowButton + username,
```

Comment:
Don't do it this way, use translation variables. You don't known how this is handled other languages.
```suggestion
      confirmButtonText: self.unfollowButton.replace('%username%', username),
```